src/arviz_plots/plots/dist_plot.py

```diff
@@ -299,6 +330,33 @@ def plot_dist(
         else:
             raise NotImplementedError("coming soon")
 
+        if face_kwargs is not False and kind in {"kde", "ecdf"}:
+            _, face_aes, face_ignore = filter_aes(
+                plot_collection, aes_by_visuals, "face", sample_dims
+            )
+            face_density = density.rename({"plot_axis": "kwarg"})
+            face_density = face_density.assign_coords(
+                kwarg=[
+                    "y_top" if coord == "y" else coord
+                    for coord in face_density.coords["kwarg"].values
+                ]
+            )
+            zeros = xr.full_like(face_density.sel(kwarg="x"), 0)
+            zeros = zeros.assign_coords(kwarg=["y_bottom"])
+            face_density = xr.concat([face_density, zeros], dim="kwarg")
```

Comment:
we can use the `.pad` method for this so we don't need to build the dataset of zeros manually:

```python
face_density = (
    density
    .rename(plot_axis="kwarg")
    .sel(kwarg=["x", "y"])
    .pad(kwarg=(0, 1), constant_value=0)
    .assign_coords(kwarg=["x", "y_top", "y_bottom"])
)
```